Scripts/build.swift

```diff
@@ -32,6 +32,7 @@ enum Platform: String, CaseIterable, CustomStringConvertible {
     case macOS_10_15
     case macOS_11
     case macOS_12
+    case macOS_13
```

Comment:
This change was just to help me run this script locally – we're not using this in CI just yet.